osc/commands_git/repo_init.py

```diff
@@ -0,0 +1,139 @@
+import sys
+
+import osc.commandline_git
+
+
+class RepoInitCommand(osc.commandline_git.GitObsCommand):
+    """
+    Init a git repo
+
+    A wrapper to git-init, plus merge .gitignore and .gitattributes from a template, plus initialize .git/config from a template
```

Comment:
This needs to be reworded, for example:
```
Initialize or update a package git repo, setting .gitignore, .gitattributes and other files from a template.
```

I need to look into initializing `.git/config` more closely.